pkg/cloud/openstack/openstack.go

```diff
@@ -1,46 +0,0 @@
-package openstack
```

Comment:
Honestly I like this approach with separate directory more then proposed one. Was there some discussion and agreement around this topic?